### PR TITLE
Use new source_bat.bash command to set env variables

### PR DESCRIPTION
**Pull request by johnwason**
I created a small script source_bat.bash to read environmental variables in batch files and export them to Git Bash. This allows for commands to be run directly in Git Bash instead of in batch files.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -40,14 +40,19 @@ install:
...
         choco uninstall mingw
+        wget https://raw.githubusercontent.com/johnwason/source_bat/master/source_bat.bash
+        source source_bat.bash 'C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat'
         choco source add -n=ros-win -s="https://roswin.azurewebsites.net/api/v2" --priority=1
         choco install ros-melodic-ros_base -y --execution-timeout=0
-        wget https://gist.githubusercontent.com/Levi-Armstrong/e3bb008e579a99c52eb74235dff6735a/raw/travis_ci_windows.bat
-        cmd "/C call travis_ci_windows.bat install "
+        #wget https://gist.githubusercontent.com/Levi-Armstrong/e3bb008e579a99c52eb74235dff6735a/raw/travis_ci_windows.bat
+        #cmd "/C call travis_ci_windows.bat install "        
+        source source_bat.bash 'c:\opt\ros\melodic\x64\setup.bat'
```
Comment (Levi-Armstrong) [inline .travis.yml]: Looks good to me. Are you wanting to keep the commented outline before merging?

File: .travis.yml [hunk 2]
```diff
@@ -40,14 +40,19 @@ install:
...
+        rosdep install --from-paths src --ignore-src -r -y
+        /c/opt/vcpkg/vcpkg.exe install jsoncpp:x64-windows
     fi
 
 script:
   - if [ "$TRAVIS_OS_NAME" != "windows" ]; then .ci_config/travis.sh; fi
-  - if [ "$TRAVIS_OS_NAME" = "windows" ]; then cmd "/C call travis_ci_windows.bat build "; fi
+  - if [ "$TRAVIS_OS_NAME" = "windows" ]; then catkin_make_isolated -DENABLE_TESTS=ON; fi
```
Comment (Levi-Armstrong) [inline .travis.yml]: I looked at industrial ci and probably want to also pass the following two cmake arguments `-DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_FLAGS_RELEASE="-O3"`

File: .travis.yml (hunk 1 above)
Comment (johnwason) [reply .travis.yml]: Comments are removed

File: .travis.yml (hunk 2 above)
Comment (johnwason) [reply .travis.yml]: I set the build type to `Release`. Setting `-O3` is unnecessary as the default flags for `Release` already do so.

**Pull request merged**